Updated changelog and notifier version (#226)
airbrake/gobrake: CHANGELOG.md

## master

## [v5.2.0][v5.2.0] (December 13, 2021)

> * Deprecating `NewMiddleware` func and replacing with `New` func in gin middleware.([#224](https://github.com/airbrake/gobrake/pull/224))

Reviewer: ```suggestion
* Deprecated `NewMiddleware` func and replaced with `New` func in gin middleware ([#224](https://github.com/airbrake/gobrake/pull/224))
```

> * Use [Apex/log](https://github.com/apex/log) severity levels instead of custom defined severity levels.([#225](https://github.com/airbrake/gobrake/pull/225))

Reviewer: ```suggestion
* Used [Apex/log](https://github.com/apex/log) severity levels instead of custom defined severity levels ([#225](https://github.com/airbrake/gobrake/pull/225))
```

## master

## [v5.2.0][v5.2.0] (December 10, 2021)

> * Refactored gin middleware([#224](https://github.com/airbrake/gobrake/pull/224))

Reviewer: I don't think this is worth mentioning in the changelog. Our users won't care about this change.

Reviewer 2: I agree with Kyrylo here. If we mention anything here it would be to say, "Deprecating `NewMiddleware` func and replacing with `New`."

We actually missed something in https://github.com/airbrake/gobrake/pull/224/files. We should have refactored `NewMiddleware` to look like this:

```
func NewMiddleware(engine *gin.Engine, notifier *gobrake.Notifier) func(c *gin.Context) {
	return New(engine, notifier)
}
```

> * Added the support for using severity levels defined by [Apex/log](https://github.com/apex/log) middleware instead of custom defined severity levels.([#225](https://github.com/airbrake/gobrake/pull/225))

Reviewer: This sentence feels wrong to me. I'll let @scottsbaldwin check it because I am not a native English speaker.

Reviewer 3: nor is he :)

Reviewer 2: ```suggestion
* Use [Apex/log](https://github.com/apex/log) severity levels instead of custom defined severity levels.([#225](https://github.com/airbrake/gobrake/pull/225))
```
